client/SDL/sdl_freerdp.cpp

```diff
@@ -303,27 +305,43 @@ static BOOL sdl_redraw(sdlContext* sdl)
 	return gdi_send_suppress_output(gdi, FALSE);
 }
 
+class EventGuard
```

Comment:
The class name seem way too generic while it's doing something very specific

Reply:
it is a private class, so no need to be universally unique.

Reply:
ok, renamed.